### DISCUSSION TITLE
Workflow error (action not allowed)

### DISCUSSION THREAD

**Discussion by sundowndev** [community/community#25129] [Actions]
<p>What is happening ?</p>
<p>I didn’t change anything to the configuration !!</p>
<p>Action tab shows <a href="https://github.com/sundowndev/underbase/commit/4a7394036141c522123ccdd05f8518bc0eb360f5/checks?check_suite_id=297367055" rel="nofollow noopener">(Unnamed workflow)</a> for all recent workflow runs since yesterday.</p>
<p><a href="https://github.com/sundowndev/underbase/actions" rel="nofollow noopener">https://github.com/sundowndev/underbase/actions</a></p>
<p>Build Failed</p>
<p>Workflow Error</p>
<p>Details</p>
<pre><code>### ERRORED 21:56:28Z

- Only actions in "sundowndev" are allowed for this repository
</code></pre>

**Comment by eddiechenms** [comment, marked as answer]
<p>The error message indicates that it is only allowed to run the actions exist within your repository or organization (sundowndev). Please check the “Settings\Actions” of your repository:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/0/023a444678a58b5ee458d23225ca744c8940a346.png" data-download-href="/uploads/short-url/jHN1ILBHLHkJincPcaXkTweszI.png?dl=1" title="1.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/56386290/181089064-08e22d2c-5054-4f48-b558-15645dd23f49.png" alt="1.png" data-base62-sha1="jHN1ILBHLHkJincPcaXkTweszI" width="690" height="364" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/0/023a444678a58b5ee458d23225ca744c8940a346_2_690x364.png, https://aws1.discourse-cdn.com/github/optimized/2X/0/023a444678a58b5ee458d23225ca744c8940a346_2_1035x546.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/0/023a444678a58b5ee458d23225ca744c8940a346_2_1380x728.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/0/023a444678a58b5ee458d23225ca744c8940a346_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">1.png</span><span class="informations">1583×836 77 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by ludwig247** [comment]
<p>Hello,</p>
<p>My job triggers on push and pull_request.</p>
<p>The settings are as stated.</p>
<p>However,  I still receive this message in case of a pull request from a forked repository. </p>
<p>Is there anything else to take care of?</p>

**Comment by emnavarro02** [comment]
<p>I have an action that references an external module.</p>
<p>Initially, the “Actions permissions” configuration in my repository was set to " <strong>Enable local Actions only for this repository</strong>" and this action was working properly but recently I noticed it stopped to work.</p>
<p>I just noticed that there is a new notification in the Github Action “Marketplace”, so I think this error was caused by this change. </p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/d/da5f338d22047fc598e482a8749a0222b4648683.jpeg" data-download-href="/uploads/short-url/v9O3vpbrDJ2FRps3NWAHJBW9RKj.jpg?dl=1" title="Capture.JPG" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/3664652/181089069-195a17d6-beb4-46bf-a27c-8701a804dada.jpeg" alt="Capture.JPG" data-base62-sha1="v9O3vpbrDJ2FRps3NWAHJBW9RKj" width="690" height="110" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/d/da5f338d22047fc598e482a8749a0222b4648683_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Capture.JPG</span><span class="informations">998×160 14 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>I had to change the “Actions permissions” configuration to " <strong>Enable local and third-party Actions for this repository</strong>" to fix it.</p>
